README.md

```diff
@@ -79,14 +79,34 @@ This will scaffold a basic workspace inside the `packages` folder.
 
 ### Publish a package
 
--   Run `pnpm build` to build the packages.
-
--   Run `pnpm changeset` then follow the instructions. This will generate a new changeset.
-
--   Run `pnpm changeset version`. This will update changed packages versions and changelogs.
-
--   Commit everything and push to your branch. Once the branch is merged into main, the package will
-    automatically get published to npm.
+The process relies 100% on the github actions workflow. PR are merged into main, which triggers a
```

Comment:
Can you detail what we should do at every step? Specifically when the changeset has to be generated, what git commands, etc.?

Reply:
yeah ! see 9b964e7b02